app/views/submissions/_publishing.html.haml

```diff
@@ -0,0 +1,6 @@
+- if submission.publishable? && submission.revocable?
+  = confirmable_action("Revoke publication", submission,
+      url: revoke_submission_path(submission), method: :patch)
+- elsif !submission.publishable?
```

Comment:
I got the impression that it was agreed between you and @nowakowski :) I believe that `#published?` makes more sense because it de facto indicates that submission's data is published and not that it can be published (without saying anything about current state).